schemas/com.snowplowanalytics.travel/add_to_itinerary/jsonschema/1-0-0

```diff
@@ -0,0 +1,43 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "type": "object",
+  "self": {
+    "vendor": "com.snowplowanalytics.travel",
```

Comment:
Is there any precedent for such a specific set of schemas in Iglu Central for demo purposes?
It feels like these don't really fit in with anything we currently have in Central, and it would be awkward if people started using them for tracking their own events and expected support from e.g. data models because they're "official" schemas.